### DISCUSSION TITLE
Feature request: Visualization of data flows of an application

### DISCUSSION THREAD

**Discussion by misch42** [Alef-Burzmali/netbox-data-flows#54] [Ideas]
hi,

I'd like to see a visualization of an application and all its data flows between the objects (source, destination). I think about about something like the topology views plugin. Is this possible?

**Comment by Alef-Burzmali** [comment]
Hello,
That's a feature I would like to add, but my competence for JavaScript and front-end development is very limited. If someone wants to help, you are welcome to open a PR.
